runtime/doc/lsp.txt

```diff
@@ -1371,11 +1372,14 @@ format({opts})                                          *vim.lsp.buf.format()*
                   (client.id) matching this field.
                 • {name}? (`string`) Restrict formatting to the client with
                   name (client.name) matching this field.
-                • {range}? (`{start:integer[],end:integer[]}`, default:
-                  current selection in visual mode, `nil` in other modes,
-                  formatting the full buffer) Range to format. Table must
-                  contain `start` and `end` keys with {row,col} tuples using
-                  (1,0) indexing.
+                • {range}?
+                  (`{start:integer[],end:integer[]}|{start:integer[],end:integer[]}[]`,
```

Comment:
Would it make more sense to change these to a tuple like `[integer, integer]`?

Reply:
I'm not 100% sure what you mean, could you give an example of what that would look Iike?

Reply:
I mean making the type of the positions a tuple instead of `integer[]`, since we're expecting their length to be 2.

Reply:
ah got it, thanks for clarifying